From 028d4517c7cf9e0f031aa71db15924c09be36cf5 Mon Sep 17 00:00:00 2001
From: Joey Hess <joeyh@joeyh.name>
Date: Sat, 7 Nov 2020 14:09:17 -0400
Subject: [PATCH] enable extensions needed by new version of persistent

Needed in order to use mkPersist in
persistent version 2.11.0.1
persistent-template version 2.9.1.0
---
 Database/ContentIdentifier.hs | 1 +
 Database/Export.hs            | 1 +
 Database/Fsck.hs              | 1 +
 Database/Keys/SQL.hs          | 1 +
 4 files changed, 4 insertions(+)

diff --git a/Database/ContentIdentifier.hs b/Database/ContentIdentifier.hs
index 0508a386e..da7f9545d 100644
--- a/Database/ContentIdentifier.hs
+++ b/Database/ContentIdentifier.hs
@@ -10,6 +10,7 @@
 {-# LANGUAGE OverloadedStrings, GADTs, FlexibleContexts, EmptyDataDecls #-}
 {-# LANGUAGE MultiParamTypeClasses, GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE DataKinds, FlexibleInstances #-}
 {-# LANGUAGE UndecidableInstances #-}
 #if MIN_VERSION_persistent_template(2,8,0)
 {-# LANGUAGE DerivingStrategies #-}
diff --git a/Database/Export.hs b/Database/Export.hs
index 1eba4dd40..99a481703 100644
--- a/Database/Export.hs
+++ b/Database/Export.hs
@@ -10,6 +10,7 @@
 {-# LANGUAGE OverloadedStrings, GADTs, FlexibleContexts #-}
 {-# LANGUAGE MultiParamTypeClasses, GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE DataKinds, FlexibleInstances #-}
 {-# LANGUAGE UndecidableInstances #-}
 #if MIN_VERSION_persistent_template(2,8,0)
 {-# LANGUAGE DerivingStrategies #-}
diff --git a/Database/Fsck.hs b/Database/Fsck.hs
index 528f3a007..c1e984197 100644
--- a/Database/Fsck.hs
+++ b/Database/Fsck.hs
@@ -9,6 +9,7 @@
 {-# LANGUAGE QuasiQuotes, TypeFamilies, TemplateHaskell #-}
 {-# LANGUAGE OverloadedStrings, GADTs, FlexibleContexts #-}
 {-# LANGUAGE MultiParamTypeClasses, GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DataKinds, FlexibleInstances #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE UndecidableInstances #-}
 #if MIN_VERSION_persistent_template(2,8,0)
diff --git a/Database/Keys/SQL.hs b/Database/Keys/SQL.hs
index f96433b36..26108b196 100644
--- a/Database/Keys/SQL.hs
+++ b/Database/Keys/SQL.hs
@@ -10,6 +10,7 @@
 {-# LANGUAGE OverloadedStrings, GADTs, FlexibleContexts #-}
 {-# LANGUAGE MultiParamTypeClasses, GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RankNTypes, ScopedTypeVariables #-}
+{-# LANGUAGE DataKinds, FlexibleInstances #-}
 {-# LANGUAGE UndecidableInstances #-}
 #if MIN_VERSION_persistent_template(2,8,0)
 {-# LANGUAGE DerivingStrategies #-}
-- 
2.28.0.297.g1956fa8f8d

